orca/orca.py

```diff
@@ -1919,7 +1919,8 @@ def run(steps, iter_vars=None, data_out=None, out_interval=1,
                     i, var))
 
         t1 = time.time()
-        for step_name in steps:
+        for j, step_name in enumerate(steps):
```

Comment:
Could easily change that code though if you prefer to keep the enumerations uniform since they're in the same loop.